### PR TITLE
Update Ngx HTML package

### PR DESCRIPTION
**Pull request by deathaxe**
1. ship ST4 syntax as of first stable release 4107.
2. add a way point for releases, requiring ST 4152.

~~requires https://github.com/princemaple/ngx-html-syntax/issues/35~~

EDIT: required tag was created.

### CODE REVIEW

File: repository/n.json [hunk 1]
```diff
@@ -624,16 +624,19 @@
 		},
 		{
 			"name": "Ngx HTML",
-			"previous_names": ["Angular2 HTML Syntax"],
```
Comment (braver) [inline repository/n.json:627]: Is this removal intentional?

Comment (princemaple) [reply repository/n.json:627]: Happy for it to be removed.
